Description: Use dynamic slot allocation for Debian as the volume & brightness
 notifications do not have the Ubuntu patches.
Author: Julian Andres Klode <jak@debian.org>
Debian-Bug: https://bugs.debian.org/550592

Index: unstable/src/defaults.c
===================================================================
--- unstable.orig/src/defaults.c	2015-05-25 16:53:52.132932955 -0400
+++ unstable/src/defaults.c	2015-05-25 16:54:03.681131343 -0400
@@ -473,8 +473,8 @@
 					  G_CALLBACK (_gravity_changed),
 					  self);
 
-	// use fixed slot-allocation for async. and sync. bubbles
-	self->slot_allocation = SLOT_ALLOCATION_FIXED;
+	// DEBIAN: Use DYNAMIC slot-allocation (0.9.23-2)
+	self->slot_allocation = SLOT_ALLOCATION_DYNAMIC;
 }
 
 static void
